sunkit_spex/legacy/thermal.py

```diff
@@ -764,15 +768,15 @@ def _warn_if_input_outside_valid_range(input_values, grid_range, param_name, par
 
 
 def _calculate_abundances(abundance_type, relative_abundances):
-    abundances = DEFAULT_ABUNDANCES[abundance_type].data
+    abundances = fetch_default_abundances(abundance_type)
```

Comment:
What about then we don't need a new function or to alter the names?
```suggestion
    abundances = DEFAULT_ABUNDANCES(abundance_type).data.copy()
```

Reply:
sure we could do a copy. the reasoning was, then whenever you reference DEFAULT_ABUNDANCES, you need to remember to do the .copy. with the function it does it by default